lib/errors.js

```diff
@@ -109,7 +108,8 @@ exports.Err = class {
 
         return format.replace(/\{\{(\!?)([^}]+)\}\}/g, ($0, isSecure, name) => {
 
-            const value = Hoek.reach(this.context, name);
+            const context = Object.assign({}, this.context, { key: this.context.label });
```

Comment:
The template language takes values from the context, the context was changed to replace `key` with `label`, which is probably incorrect, but the template language didn't automagically transform `key` into `label`. I maintain that a proper fix needs to be a breaking change.